scalableminds/webknossos
# Avoid Java Temp Files in task creation

- During task creation, we no longer use java’s own temporary file code (see reasons in linked issue below), but instead use a custom temporary file service.
- The initial data requests are no longer sent in parallel to the tracingstore but instead sequentially, to avoid overloading the tracingstore (new function Fox.serialSequenceBox)

### URL of deployed dev instance (used for testing):
- https://taskcreationnotempfiles.webknossos.xyz

### Steps to test:
- Create volume or hybrid task type
- try to create (hundreds of) tasks with initial volume data
- should not fail

### Issues:
- fixes #6173

------
